LABELING.md

```diff
@@ -0,0 +1,51 @@
+
+# Labeling Guidelines
+
+Now that the screenshots are available, they need to be labeled. The labeling phase operates on couples of comparable screenshots.
+
+## Images marked as compatible - y 
+---
+#### Couples of images that are clearly compatible.
+#### They look the same.
+#### firefox\_chrome\_overlay window should nearly overlap them.
+---
+<p align="center"><img src="labeling_guide/y1.png" width=617 height=357></p>
+<p align="center"><img src="labeling_guide/y2.png" width=617 height=357></p>
+<p align="center"><img src="labeling_guide/y3.png" width=617 height=357></p>
+<p align="center"><img src="labeling_guide/y4.png" width=617 height=357></p>
+<p align="center"><img src="labeling_guide/y5.png" width=617 height=357></p>
+
+## Bounding boxes marked as incompatible - n 
+---
+#### Couples of images which are not compatible
+#### They are different.
+#### Mark the parts which are logically different.
+> Improper loading of images, missing text, different design, different languages are marked incompatible.
+---
+
+## Bounding boxes marked as different yet compatible - d 
+---
+#### Couples of images that are compatible, but with content differences.
+#### They look different.
+#### Mark the parts which are logically the same.
+>Different advertisements, different videos loaded, time-in-clock, captchas are marked
+as different yet compatible.
```

Comment:
Also add the example which was in the readme about the news site.

`Different videos loaded` shouldn't be there, the same video should be loaded in both browsers if the site is compatible. You can say `different frames of a video loaded`.
Rename `time-in-clock` as it is unclear.

I would say something like `Since the screenshots are taken at different times in the two browsers, there are differences which are not incompatibilities but are actually due to the different timing. For example, a banner could be showing a different advertisement, a video could be in two different frames, a clock could be showing different time, a captcha could be showing different characters or images, two news could be different, and so on`.